src/main/java/Duke.java

```diff
@@ -1,10 +1,109 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    private static Task[] tasks=new Task[100];;
```

Comment:
Maybe the = sign could be separated with spaces. For eg `private static Task[] = new Task[100];`